azkaban-common/src/test/java/azkaban/trigger/BasicTimeCheckerTest.java

```diff
@@ -77,6 +76,221 @@ public void basicTimerTest() {
     }
 
     assertTrue(cond.isMet());
+  }
+
+  /**
+   * Test Base Cron Functionality.
+   */
+  @Test
+  public void cronTimerTestCurrentZone() {
```

Comment:
@suvodeep-pyne Thanks for your review! I believe we might have this to keep readability.